### DISCUSSION TITLE
expected function, found `CapsuleReader<'_, '_>

### DISCUSSION THREAD

**Discussion by busslina** [GregoryConrad/rearch-rs#61] [General]
I'm using `get` exactly as shown on the README example, but got this error:

![imagen](https://github.com/user-attachments/assets/5b4c2102-6636-4ab7-af2f-e6a6feb21fc0)

**Comment by busslina** [comment]
BTW, in the readme it should be `register.register` instead of `register`. Or maybe I'm missing a feature/import. VSCode is not suggesting me any import on get/register.

**Reply by GregoryConrad** [reply]
`get()` and `register` are behind a feature flag since they require nightly. Until they're stabilized (which will be years), you can do `get.as_ref(capsule)` and `register.register()`

**Reply by GregoryConrad** [reply]
There's a note in the readme about needing the feature flag in order to get that better syntax

**Reply by busslina** [reply]
Cool, thanks!
